Yank/tests/test_repex.py

```diff
@@ -324,7 +324,7 @@ def decompact_state_variable(variable):
                 unsampled_serialized.append(unsampled_dict)
 
             # Two of the three ThermodynamicStates are compatible.
-            assert isinstance(states_serialized[0]['standard_system'], str)
+            assert 'standard_system' in states_serialized[0]
```

Comment:
Actually we pop `'standard_system'` from the `dict` when we add `'_Reporter__compatible_state'` (see [here](https://github.com/choderalab/yank/blob/master/Yank/repex.py#L421)). I got it wrong the first time.